addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -3047,7 +3056,10 @@ var ThreadWidget = Widget.extend({
             } else {
                 displayAuthorMessages[message.getID()] = !options.squashCloseMessages;
             }
-            prevMessage = message;
+            // ignore removed message to consider as previous message
+            if (!message.isEmpty()) {
```

Comment:
I feel like this check is done a bit too late. Rather than adding this condition here, you may want to filter `messages` to not include empty messages.

Reply:
I'm surprised, `isEmpty` was already defined on legacy message?

Reply:
Looks like it was https://github.com/odoo-dev/odoo/blob/15.0-livechat-adapted-recent-changes-fix-bit/addons/im_livechat/static/src/legacy/public_livechat.js#L1875=